style.css

```diff
@@ -146,3 +145,228 @@ li {
   margin: 0 auto;
   border-radius: 24px;
 }
+
+.portfolio-section {
+  padding: 16px;
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+}
+
+.portfolio-product {
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  width: 367px;
+  margin: 20px auto;
+}
+
+.portfolio-img {
+  width: 100%;
+  border-radius: 8px 8px 0 0;
+  height: 300px;
+  background-color: #ebf0ee;
+}
+
+.portfolio-ul {
+  padding: 0;
+  display: flex;
+  text-align: center;
+  list-style-type: none;
+  justify-content: space-between;
+  margin-top: 16px;
+  gap: 12px;
+}
+
+.portfolio-elements {
+  display: flex;
+  flex-direction: column;
+  border-radius: inherit;
+  padding: 30px 12px;
+  justify-content: center;
+  align-items: center;
+}
+
+.portfolio-elements h2 {
+  color: #3a4a42;
+}
+
+.portfolio-li {
+  padding: 8px 12px;
+  font-weight: 600;
+  font-size: 12px;
+  background-color: #ebf0ee;
+  border-radius: 5px;
+  align-content: end;
+  color: #3a4a42;
+  line-height: 16px;
+}
+
+button {
+  font-family: 'Inter', sans-serif;
+  border: none;
+  font-size: 17px;
+  font-weight: 500;
+  cursor: pointer;
+}
+
+.portfolio-button {
+  width: 122px;
+  height: 48px;
+  margin-top: 24px;
+  padding: 12px;
+  background-color: #36b37f;
+  color: #fff;
+  border-radius: 4px;
+  text-align: center;
+}
+
+a:visited {
+  color: #fff;
+}
+
+.portfolio-button:hover {
+  background-color: #36b37f;
+  box-shadow: 0 8px 16px rgb(54, 181, 128, 0.24);
+}
+
+.portfolio-btn:active {
+  background-color: #008552;
+}
+
+.about-me {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  background-image: url(images/illustration-about\ me-1-desktop.svg);
+  background-repeat: no-repeat;
+  background-position: right top;
+  position: relative;
+  color: #172b4d;
+}
+
+.about-me .about-me-img {
+  position: absolute;
+  background-repeat: no-repeat;
+  background-position: left top;
+  bottom: 0;
+  left: 0;
+}
+
+h3 {
+  font-size: 24px;
+  font-weight: 600;
+  margin-top: 16px;
+}
+
+.about-me > h3 {
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  margin-bottom: 24px;
+  margin-top: 68px;
+}
+
+.about-me > p {
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  margin-bottom: 36px;
+}
+
+.about-me .description {
+  color: #42526e;
+}
+
+.resume-btn {
+  display: flex;
+  margin-top: 24px;
+  padding: 12px;
+  background-color: #36b37f;
+  color: #fff;
+  border-radius: 5px;
+  border: 1px solid #36b37f;
+  justify-content: center;
+}
+
+.portfolio-lang {
+  width: 365px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+  margin: 20px auto;
+}
+
+.portfolio-globe {
+  display: flex;
+  flex-direction: column;
```

Comment:
- [ ] You seem to have repeated this specific combination of styling multiple times in your CSS file. In following with DRY (Don't Repeat Yourself) principles, please extract these styles into a separate class and then add them as class attributes in the respective HTML tags. For example:
```
.d-flex {
  display: flex;
}
.flex-col {
  flex-direction: column;
}
```

You can then add `d-flex` and `flex-col` as classes in your HTML tags where appropriate